recipe/meta.yaml

```diff
@@ -11,47 +11,68 @@ source:
   sha256: 4fd2eb3736cea503be94ab18d31a669addbaf2dd6a8249527cef94163bb93975
 
 build:
+  number: 4
   noarch: python
-  number: 3
-  script: python -m pip install . -vv
 
-requirements:
-  host:
-    - hatchling
-    - pip
-    - python {{ python_min }}
+outputs:
+  - name: osmnx-base
+    script: build_base.sh
+    build:
+      noarch: python
+    requirements:
+      build:
+        - python                              # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+      host:
+        - python {{ python_min }}
+        - hatchling
+        - pip
+      run:
+        - python >={{ python_min }}
+        - geopandas-base >=1.0
+        - pyproj >=3.3.0
+        - pyogrio >=0.7.2
```

Comment:
Let's just continue using the `geopandas` dependency rather than cherry-picking sub-dependencies from `base` (whose min values we'll need to keep track of and maintain here).

Reply:
Right, I understand your concern. The main reason that I added `geopandas` is that it adds several additional deps that I don't think are needed: `matplotlib-base`, `mapclassify`, `folium`, and `xyzservices`. They are needed mainly for the `explore` method of `geopandas`.

Reply:
What about keeping this for the `osmnx-base` output and add `geopandas` to the `osmnx` output? This will be used by maintainers anyway, and they can adjust the min versions in their own recipe if needed.

Reply:
I think this would be an issue to raise upstream at the GeoPandas feedstock if `geopandas-base` could be made slimmer by avoiding optional dependencies only needed by the `explore` method. But for our feedstock here, I don't think we want to get into the game of cherry-picking and maintaining min version pins for upstream dependencies' dependencies.

Reply:
Sounds good, I made the necessary changes.